dt-login/login-shortcodes.php

```diff
@@ -12,11 +12,14 @@
 function dt_firebase_login_ui( $atts ) {
 
     $default_lang = 'en';
+    $default_redirect_to = DT_Login_Fields::get( 'redirect_url' );
     $atts = shortcode_atts( [
         'lang_code' => $default_lang,
+        'redirect_to' => $default_redirect_to,
     ], $atts );
 
     $lang_code = $atts['lang_code'];
+    $redirect_to = $lang_code;
```

Comment:
@squigglybob i'm not sure i follow the logic here.
The redirect will *always* be the lang code?

Reply:
😆 doh. That is not what's it's meant to be, and I have no idea how it was even working!!!